code/script.js

```diff
@@ -0,0 +1,56 @@
+var flag = false;
+
+// displays numerous inputs (numbers) on the screen
+function displaynumber(n1) {
```

Comment:
n1 ... what is is? Please use some correct name what n1 represented because this is magic.

Reply:
n1 changed to what it represents meaning inputValue same done with n2 goes inputValue. As you said 1 comment 1 commit so hope this one is better then previous one

Reply:
Yes, but ... Please check it in all places.
I see still some e, b, ... etc

Reply:
yes will do, I just follow the instruction, e, b, ... etc is mentioned in your comment below, so it is gonna be another commit...